src/module/actor/sheet/item-summary-renderer.ts

```diff
@@ -195,6 +195,14 @@ export class CreatureSheetItemRenderer<AType extends CreaturePF2e> extends ItemS
                 }
 
                 if (actor instanceof CharacterPF2e) {
+                    if (chatData.variants) {
+                        buttons.append(
+                            `<span><button class="spell_attack" data-action="selectVariant">${game.i18n.localize(
+                                "PF2E.SpellVariantSelectVariantLabel"
+                            )}</button></span>`
```

Comment:
Avoid doing so much in a string interpolation that it requires multiple lines: localize the label above it and interpolate that.